Code/GraphMol/FileParsers/SequenceParsers.cpp

```diff
@@ -1187,6 +1187,7 @@ static const char *LookupHELMPeptideMonomer(const char *ptr) {
       break;
     case 'D':
       if (ptr[1] == '\0') return "ASP";
+      break;

```

Comment:
Though it's clearly stylistically correct, I don't think this actually does anything.